### PR TITLE
Doc house keeping

### PR DESCRIPTION
**Pull request by KrisNey-MSFT**
Added text for Slow Path & Fast Path

### CODE REVIEW

File: Documentation/sdn-features-packet-transforms.md [hunk 1]
```diff
@@ -217,6 +217,8 @@ Etc…
 
 
  ## Packet Flow
+For the first packet of a TCP flow, we take the Slow Path, running the transposition engine and matching at each layer.  For subsequent packets, we take the Fast Path, 
+matching a unified flow via UFID and applying a transposition directly against rules.
```
Comment (chrispsommers) [inline Documentation/sdn-features-packet-transforms.md:221]: What is a UFID? If it's a MSFT-specific buzzword, might want to reword it to be generally meaningful. Should be in glossary.

**Review by chrispsommers — CHANGES_REQUESTED** [review]

File: Documentation/sdn-features-packet-transforms.md (hunk 1 above)
Comment (KrisNey-MSFT) [reply Documentation/sdn-features-packet-transforms.md:221]: UFID is a Unified Flow ID.  Can put into the Glossary.

**Review by mmiele — COMMENTED** [review]
Formatted **Counters** section.
